.github/workflows/master-build.yml

```diff
@@ -336,3 +339,82 @@ jobs:
           pushd .gotmp/bin/windows_amd64/chocolatey
           docker run --rm -v $PWD:/tmp/chocolatey -w /tmp/chocolatey linuturk/mono-choco push -s https://push.chocolatey.org/ --api-key "${CHOCOLATEY_API_KEY}"
           popd
+
+  push-devcontainers:
+    name: Push devcontainer feature
+    runs-on: ubuntu-latest
+    needs: artifacts
+    steps:
+      - name: Load 1password secret(s)
+        uses: 1password/load-secrets-action@v2
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: "${{ secrets.PUSH_SERVICE_ACCOUNT_TOKEN }}"
+          GITHUB_TOKEN: "op://push-secrets/DDEV_GITHUB_TOKEN/credential"
+
+      - uses: actions/checkout@v4
+        with:
+          # We need to get all branches and tags for git describe to work properly
+          fetch-depth: 0
+
+      - name: Replace version in devcontainer-feature.json
+        run: |
+          sed -i 's/\${DDEV_VERSION}/'"${GITHUB_REF##*/}"'/g' ./containers/devcontainers/install-ddev/devcontainer-feature.json
```

Comment:
`${GITHUB_REF##*/}` looks the same as `${GITHUB_REF_NAME}` from https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables